integration/nwo/orderer_template.go

```diff
@@ -35,7 +35,7 @@ General:
     ServerMinInterval: 60s
     ServerInterval: 7200s
     ServerTimeout: 20s
-  BootstrapMethod: file
+  BootstrapMethod: {{ .BootstrapMethod}}
```

Comment:
nit: white space between the variable and the closing `}}`

Reply:
Done